docs/getting-started-guides/ubuntu/operational-considerations.md

```diff
@@ -116,45 +116,52 @@ juju switch default
 
 ### Running privileged containers
 
-By default juju-deployed clusters do not support running privileged containers. If you need them, you have to edit ```/etc/default/kube-apiserver``` on the master nodes, and ```/etc/default/kubelet``` on your worker nodes. 
+By default juju-deployed clusters do not support running privileged containers.
+If you need them, you have to enable the ```allow-privileged``` config on both
+kubernetes-master and kubernetes-worker:
 
-On Kubernetes Core or on small deployment, run the following commands from the Juju client: 
+```
+juju config kubernetes-master allow-privileged=true
+juju config kubernetes-worker allow-privileged=true
+```
 
-#### Manually 
+### Private registry
 
-1. Update the Master
+With the "registry" action that is part for the kubernetes-worker charm, you
+can very easily create a private docker registry, with authentication, and
+available over TLS. Please note that the registry deployed with the action is
+not HA, and uses storage tied to the kubernetes node where the pod is running.
+So if the registry pod changes is migrated from one node to another for
+whatever reason, you will need to re-publish the images.
 
-```
-juju ssh kubernetes-master/0 "sudo sed -i 's/KUBE_API_ARGS=\"/KUBE_API_ARGS=\"--allow-privileged\ /' /etc/default/kube-apiserver && sudo systemctl restart kube-apiserver.service"
-```
+#### Example usage
 
-2. Update the Worker(s)
+Create the relevant authentication files. Let's say you want user ```userA```
+to authenticate with the password ```passwordA```. Then you'll do:
 
 ```
-juju ssh kubernetes-worker/0 "sudo sed -i 's/KUBELET_ARGS=\"/KUBELET_ARGS=\"--allow-privileged\ /' /etc/default/kubelet && sudo systemctl restart kubelet.service"
+echo "userA:passwordA" > htpasswd-plain
+htpasswd -c -b -B htpasswd userA passwordA
 ```
 
-#### Programmatically
-
-If the deployment is larger the following commands will run on all units successively: 
+(the `htpasswd` program comes with the ```apache2-utils``` package)
 
-1. Update all Masters
+Supposing your registry will be reachable at ```myregistry.company.com```, and
```

Comment:
I think "will be reachable" is correct since the registry doesn't exist until after the `registry` action is run. Gonna stick with that unless you have further objections :)